### DISCUSSION TITLE
Export CSV based on Filter

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#8383] [Help] [answered]
Hello,
Is it possible to export data as CSV based on Filter. I create a filter in my database and then i export data into CSV but it exported all data not based on filter.

**Comment by melohagan** [comment, marked as answer]
Hey @muksidin 

Assuming you are using the Internal Budibase DB, then you can create a view which has a filter, and then export the rows of your view: https://docs.budibase.com/docs/views

**Reply by muksidin** [reply]
Okay i think i can export csv file in main database. thanks @melohagan for your help.
